chain/bitcoind_conn.go

```diff
@@ -164,6 +186,21 @@ func (c *BitcoindConn) blockEventHandler() {
 	log.Info("Started listening for bitcoind block notifications via ZMQ "+
 		"on", c.zmqBlockConn.RemoteAddr())
 
+	// Set up the buffers we expect our messages to consume. ZMQ
+	// messages from bitcoind include three parts: the command, the
+	// data, and the sequence number.
+	//
+	// We'll allocate a fixed data slice that we'll reuse when reading
+	// blocks from bitcoind through ZMQ. There's no need to recycle this
+	// slice (zero out) after using it, as further reads will overwrite the
+	// slice and we'll only be deserializing the bytes needed.
+	var (
+		command [len(rawBlockZMQCommand)]byte
+		seqNum  [seqNumLen]byte
+		data    = make([]byte, maxRawBlockSize)
+		bufs    = [][]byte{command[:], data, seqNum[:]}
```

Comment:
If I’m not mistaken, `bufs` needs to be reinstantiated with every for loop, otherwise the slices can be shortened over time

Reply:
Yep, fixed.